src/components/UserActions/UserActivity.ts

```diff
@@ -59,6 +66,10 @@ export class UserActivity extends Component {
     static readonly ID = 'UserActivity';
     static readonly options: IUserActivityOptions = {
         userId: ComponentOptions.buildStringOption({ required: true }),
+        customActionsExclude: ComponentOptions.buildListOption({
+            defaultValue: ['ticket_create_start', 'ticket_field_update', 'ticket_next_stage', 'ticket_classification_click'],
```

Comment:
Could be extracted in an export const instead for the list of defaultValues, depends on what you prefer.

Reply:
meh, they are nice and close to where they are actually used, it's fine to me I think?